base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
The problem with that is that `pipe` with more than one argument is a lazy constructor for a command object, while `Pipe()` creates a pipe object that `run` modifies.